### PR TITLE
Updating Customer Feedback section

### PR DESCRIPTION
**Pull request by emilyhollinger**

### CODE REVIEW

File: operations/customer-success/README.md [hunk 1]
```diff
@@ -2,8 +2,38 @@
 
 ## Customer Success
 
-### Customer Success Key Info
+### Customer Feedback
+
+## Overview
+Customer Feedback calls are conducted by all customer facing employees at Mattermost. The objective of a Customer Feedback call is to gather the following information from a customer.
```
Comment (justinegeffen) [inline operations/customer-success/README.md:8]:
```suggestion
Customer Feedback calls are conducted by all customer-facing employees at Mattermost. The objective of a Customer Feedback call is to gather the following information:
```

File: operations/customer-success/README.md [hunk 2]
```diff
@@ -2,8 +2,38 @@
...
+
+- Feature requests
+- Integration usage
+- Use cases
+- Understand which teams use Mattermost
+- Product design review feedback
+- Challenges solving
```
Comment (justinegeffen) [inline operations/customer-success/README.md:15]:
```suggestion
- Challenges faced
```

File: operations/customer-success/README.md [hunk 3]
```diff
@@ -2,8 +2,38 @@
...
+
+These calls are logged in Salesforce for tracking and reporting purposes.  
+
+## Logging calls in Salesforce
+
+1. Log into Salesforce
+2. Type in account name in the search bar located in the middle of the screen
+3. Select the account
+4. Hover over activity history
```
Comment (justinegeffen) [inline operations/customer-success/README.md]:
```suggestion
4. Hover over **Activity History**
```

File: operations/customer-success/README.md [hunk 4]
```diff
@@ -2,8 +2,38 @@
...
+5. Select “log a call” in the upper right-hand side of the screen
```
Comment (justinegeffen) [inline operations/customer-success/README.md]:
```suggestion
5. Select **Log a Call** in the top-right corner of the screen
```

File: operations/customer-success/README.md [hunk 5]
```diff
@@ -2,8 +2,38 @@
...
+6. Fill out the following sections
+   Subject - Quick description of the call.  IE: Customer Reply Thread Design Review
```
Comment (justinegeffen) [inline operations/customer-success/README.md:27]:
```suggestion
   **Subject:** Short description of the call (e.g. "Customer Reply Thread Design Review")
```

File: operations/customer-success/README.md [hunk 6]
```diff
@@ -2,8 +2,38 @@
...
+   Name: Customer contact(s) that attended the call
```
Comment (justinegeffen) [inline operations/customer-success/README.md:28]:
```suggestion
   **Name:** Customer contact(s) who attended the call
```

File: operations/customer-success/README.md [hunk 7]
```diff
@@ -2,8 +2,38 @@
...
+   Type: Was this a call or meeting (in-person)
```
Comment (justinegeffen) [inline operations/customer-success/README.md:29]:
```suggestion
   **Type:** Whether this was a call or in-person meeting
```

File: operations/customer-success/README.md [hunk 8]
```diff
@@ -2,8 +2,38 @@
...
+   Related To: Account name
```
Comment (justinegeffen) [inline operations/customer-success/README.md:30]:
```suggestion
   **Related To:** Account name
```

File: operations/customer-success/README.md [hunk 9]
```diff
@@ -2,8 +2,38 @@
...
+   Sub-Type: Choose "Customer Feedback"
```
Comment (justinegeffen) [inline operations/customer-success/README.md:31]:
```suggestion
   **Sub-Type:** Choose **Customer Feedback**
```

File: operations/customer-success/README.md [hunk 10]
```diff
@@ -2,8 +2,38 @@
...
+   Due Date: Auto-populated to today's date
```
Comment (justinegeffen) [inline operations/customer-success/README.md:32]:
```suggestion
   **Due Date:** Auto-populated to today's date
```

File: operations/customer-success/README.md [hunk 11]
```diff
@@ -2,8 +2,38 @@
...
+   Comments: Details of the call or meeting
```
Comment (justinegeffen) [inline operations/customer-success/README.md:33]:
```suggestion
   **Comments:** Details of the call or meeting
```

File: operations/customer-success/README.md [hunk 12]
```diff
@@ -2,8 +2,38 @@
...
+ 7. Select Save
```
Comment (justinegeffen) [inline operations/customer-success/README.md]:
```suggestion
 7. Select **Save**
```

**Comment by justinegeffen** [comment]
Note on the changes:
- I've added formatting to the UI elements and included the punctuation (colons) inside the formatting as I assumed the formatting in SF includes the colon in the menu name. If that is incorrect, please LMK and I'll update it. 
- I've added periods to the list items and committed that change.

**Review by emilyhollinger — COMMENTED** [review]
Thanks @justinegeffen the changes look good

**Review by justinegeffen — APPROVED** [review]

**Review by rbradleyhaas — APPROVED** [review]

**Pull request merged**